doc/onboarding.md

```diff
@@ -169,6 +169,7 @@ Landing a PR
     * `Reviewed-By: human <email>`
       * Easiest to use `git log` then do a search
       * (`/Name` + `enter` (+ `n` as much as you need to) in vim)
+      * Only include collaborators who have commented "LGTM"
```

Comment:
Oh good point! I hadn't considered that. It might be worthwhile clarifying
that in the readme, I think :)

On Saturday, August 6, 2016, Yorkie Liu <notifications@github.com> wrote:

> In doc/onboarding.md
> <https://github.com/nodejs/node/pull/7183#discussion_r73786660>:
>
> > @@ -169,6 +169,7 @@ Landing a PR
> >      * `Reviewed-By: human <email>`
> >        * Easiest to use `git log` then do a search
> >        * (`/Name` + `enter` (+ `n` as much as you need to) in vim)
> > +      * Only include collaborators who have commented "LGTM"
>
> But currently the CTC members & collaborators are different lists in our README.md#Current
> Project Team Members
> <https://github.com/nodejs/node#current-project-team-members>, I thought
> that might be have something confused to new comers :-)
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/nodejs/node/pull/7183/files/f645b4c8da57891c7cf5b59987548e87e32479fb#r73786660>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAa2eQ5zIYoRSJARismkpHzT-BoE0xn2ks5qdJhUgaJpZM4Ivauh>
> .
>